.golangci-kal.yml

```diff
@@ -23,6 +23,7 @@ linters:
               - "nofloats" # Ensure floats are not used.
               - "optionalorrequired" # Every field should be marked as `+optional` or `+required`.
               - "requiredfields" # Required fields should not be pointers, and should not have `omitempty`.
+              - "statusoptional" # Ensure all first children within status should be optional.
```

Comment:
Can't believe we have no findings :)

Does someone want to take a quick look if the linter has some gaps? (by checking our status structs)